index.rst

```diff
@@ -1,57 +1,343 @@
+    - :ref:`ScriptQueue <Script-Queue>`
+    - :ref:`Jupyter notebooks <Jupyter-notebooks>`
+
+This document begins with a high-level overview of the three components and explores how users can take advantage of their features to accomplish different types of tasks.
+Then, we define three main actors and outline their roles and assumed levels of knowledge regarding when performing specific tasks.
+These actors and roles are used to assist in defining the usability expectations of the components and control system in general as well as the underlying assumptions of the knowledge required for each role.
+In reality, the people's roles will be less discrete and there will be significant overlap between these roles.
+The identified actors are:
+
+    1. Observing Specialists
+    2. Commissioning Scientists
+    3. Software Developers
+
+Finally, we present concrete examples of interactions that the actors will perform with the OCS, linking to procedures on how to do them using the multiple components.
+These examples explore how users best take advantage of the component's features.
+They also include cases of how *not* to use a component due limitations or inefficiencies in various areas.
+The examples are also useful in facilitating discussion regarding the ease-of-use of each operation, which is expected to stimulate discussion on how to best improve system usability and/or performance.
+
+.. _LOVE:
+
+LOVE
+====
+
+Tiago insert (or link) content here.
+
+Important to say that users will interact with basic observatory operations via LOVE, but what is actually happening on the back end is that these operations (e.g. offsets) are actually scripts that get loaded and performed.
+
+
+.. _Script-Queue:
+
+Script Queue
+============
+
+The `ScriptQueue`_ is the main hub for the execution of regular operations and requires a minimum level of interaction by the user.
+As mentioned in the previous section, it is also used as a back-end for user-commanded actions performed using the `LOVE`_ interface.
+For example, when a user performs a telescope offset operation using LOVE, the actual action perform by LOVE system will the queueing of a :ref:`SAL Script <SAL_Scripts>` that then gets executed by the `ScriptQueue`_.
+The ScriptQueue is also the main tool observers will rely on to perform regular nighttime operations, such as tracking a target and obtaining a standardized set of observations.
+The `ScriptQueue`_ is flexible in how it handles and executes scripts but generally it runs sequentially.
+As soon as one script finishes, the next one begins until no scripts remain to be run.
+
 ..
-  Technote content.
+    This next bit feel repetitive..
 
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
+This architecture (or methodology) means the number functions that get executed by the scriptQueue will be very large and their complexity varies significantly.
```

Comment:
"the number functions" -> "the number of functions" ?

Not sure "functions" is the correct term here as it may be confused by functions in programing. Maybe "the number of operations"?

Reply:
replaced with "number of different operations"